contrib/guix/guix-build.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+export LC_ALL=C
+
+set -ex
+
+# Download the depends sources now as we won't have internet access in the build
+# container
+make -C depends -j"$(nproc)" download ${SOURCES_PATH+SOURCES_PATH="$SOURCES_PATH"}
+
+
+for host in ${HOSTS=i686-linux-gnu x86_64-linux-gnu arm-linux-gnueabihf aarch64-linux-gnu riscv64-linux-gnu}
+do
+    # Run the build script 'contrib/guix/build.sh' in the build container
+    # specified by 'contrib/guix/manifest.scm'
+    guix environment --manifest=contrib/guix/manifest.scm \
+                     --load-path=contrib/guix/packages \
+                     --container \
+                     --pure \
+                     --no-cwd \
```

Comment:
See my https://github.com/bitcoin/bitcoin/pull/15277#issuecomment-507046880

Reply:
Hey @hebasto! I think you need a:
```
guix pull --url=https://github.com/dongcarl/guix.git \
          --branch=2019-05-bitcoin-staging
```